src/main/java/com/selfxdsd/todocli/JsonTodo.java

```diff
@@ -17,7 +17,7 @@ public JsonTodo(final Todo todo) {
                 .add("id", todo.getID())
                 .add("start", todo.getStart())
                 .add("end", todo.getEnd())
-                .add("ticketID", todo.getTicketID())
+                .add("originatingTicket", todo.getTicketID())
```

Comment:
@amihaiemil Looks like `JsonTodo` was not updated in #22, I've updated here.
Oh, and any reason why we don't map `file` field in `JsonTodo`?

Reply:
@criske not really, the file should also be present in JsonTodo. Can you add it please? :D